agent/internal/task_config.go

```diff
@@ -87,7 +87,9 @@ func NewTaskConfig(d *apimodels.DistroView, p *model.Project, t *task.Task, r *m
 		Task:         t,
 		BuildVariant: bv,
 		Expansions:   &e,
-		WorkDir:      d.WorkDir,
+	}
+	if d != nil {
+		taskConfig.WorkDir = d.WorkDir
```

Comment:
I think this could just be set to the agent working directory. In doing so, we could also remove `(DistroView).WorkDir` field entirely because it's the same as the agent working directory in all cases anyways.

Reply:
Sounds good